scss/mixins.scss

```diff
@@ -20,262 +17,106 @@ $_o-brand-default-not-set-warning: false !default; // Has the default not set wa
     @if type-of($brand) != 'string' or $brand == '' {
         @error 'Could not set brand "#{$brand}" for component "#{$component}". A brand of type string must be given.';
     }
-    // Define brand configuration.
-    @if not _oBrandIsDefined($component, $brand) {
-        $result: _oBrandUpdateConfig($component, $brand, $config);
+    // Do not allow a component brand to be overridden.
```

Comment:
I think having a list of brands to compare against will help with validating `$brand`, we will catch situations such as a leading or trailing space (`$brand: 'masterbrand ';`)